apps/examples/st_things/light/sample_light_user.c

```diff
@@ -142,3 +148,16 @@ bool handle_set_request_on_ct(st_things_set_request_message_s *req_msg, st_thing
 	printf("[%s]OUT-handle_set_request_on_ct() called..\n", TAG);
 	return true;
 }
+
+void set_rep_push_mesg(void)
+{
+	st_things_representation_s *st_things_rep = st_things_create_representation_inst();
+	if (st_things_rep == NULL) {
+		printf("[%s]Failed to create st_things representation.\n", TAG);
+		return;
+	}
+	st_things_rep->set_str_value(st_things_rep, SEC_ATTRIBUTE_PUSH_MESG_SOURCE, "/alarms/vs/0");
+	st_things_rep->set_str_value(st_things_rep, SEC_ATTRIBUTE_PUSH_MESG_TYPE, "4");
+	st_things_rep->set_str_value(st_things_rep, SEC_ATTRIBUTE_PUSH_MESG_CODE, "Light 50 % progress");
+	printf("St_things send push notifiation to cloud!! %d\n", st_things_push_notification_to_cloud(g_target_uri, st_things_rep));
```

Comment:
In printf, instead of st_things, better to use the macro "TAG".

Reply:
fixed.